src/main/java/rockset/RocksetConnectorConfig.java

```diff
@@ -64,6 +65,14 @@ public static ConfigDef config() {
                 .build()
         )
 
+        .define(
```

Comment:
Will this hold for too long if the batch is never full? Do we have some config to make sure we flush after certain time period

Reply:
It will still flush promptly from my understanding. This just caps how large a single request can be.

Reply:
@abbccdda here is the logic: https://github.com/rockset/kafka-connect-rockset/blob/af273f4851876d1ce3804a6028a0be8be7d4f372/src/main/java/rockset/RocksetRequestWrapper.java#L81 IIUC. Batch size only ensures that we limit POST message body size.